### ISSUE TITLE
Purging removes github submodules

### ISSUE THREAD

**Issue by MarcCoquand** [ChrisPenner/SitePipe#21]
Hello, whenever the site is created it purges the 'dist' folder. This makes it difficult to incorporate it into git submodules since it removes the metadata. Git submodules are important if you want the compiled website to show up on your master page for github pages.

**Comment by ChrisPenner** [comment]
Good point! I think `slick` would handle this better, but it's probably a good idea to add an option to not wipe everything out every rebuild.
